app/views/users/loginpage.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_for(:user, url: login_path, method: :post) do |f| %>
+  <a><%= f.label :name %></a>
+  <a><%= f.text_field :name %></a>
```

Comment:
for some reason these anchors aren't showing up in black font, I tried using a form class and 
.form {
color: black;
}       to no avail

Reply:
you need to define the class `class=` in html and `class:` in erb